arrow-weak/src/test/kotlin/arrow/weak/WeakTest.kt

```diff
@@ -27,7 +27,7 @@ class WeakTest : UnitSpec() {
         }
 
         testLaws(
-            EqLaws.laws { Weak(it) },
+            EqLaws.laws { Weak(it) }

```

Comment:
I believe Pablo just removed this module altogether for now, so that'd fix your merge.